src/event/model/event.rs

```diff
@@ -87,10 +96,12 @@ impl Event {
             "#,
             id
         )
-        .fetch_all(pool);
+        .fetch_all(&mut transaction)
+        .await
+        .map_err(|inner| EventError::DatabaseError { inner })?;
 
-        let (string_parameters, uuid_parameters) = try_join!(string_parameters, uuid_parameters)
-            .map_err(|inner| EventError::DatabaseError { inner })?;
+        // let (string_parameters, uuid_parameters) = try_join!(string_parameters, uuid_parameters)
+        //     .map_err(|inner| EventError::DatabaseError { inner })?;
```

Comment:
Open TODO: need to review whether we can still use `join` with this pattern (so that stuff that can be executed in parallel will be executed in parallel). Problem here at the moment is the mutable reference...